Rockerfile.mesa

```diff
@@ -29,31 +31,73 @@ FROM {{ $image }}:llvm-{{ .LLVM }}
 
 LABEL maintainer "Juan A. Suarez Romero <jasuarez@igalia.com>"
 
-RUN apt-get update                                   \
-  && apt-get -y --no-install-recommends install git  \
-     {{ if eq .BUILD "windows" }}mingw-w64{{ end }}  \
+RUN apt-get update                                      \
+  && apt-get -y --no-install-recommends install git     \
+     {{ if eq .BUILD "windows" }}mingw-w64{{ end }}     \
+     {{ if eq .BUILD "android" }}                       \
+       wget openjdk-8-jdk python sudo make gcc gettext  \
+       python-mako bison libc6-dev xz-utils             \
+     {{ end }}                                          \
   && rm -fr /var/lib/apt/lists/*
 
+{{ if eq .BUILD "android" }}
+RUN adduser --gecos "" local && passwd -d local && adduser local sudo
+{{ end }}
 USER local
 
+WORKDIR /home/local
+
 {{ if .Env.MAKEFLAGS }}
 ENV MAKEFLAGS={{ .Env.MAKEFLAGS }}
 ENV SCONSFLAGS={{ .Env.MAKEFLAGS }}
 {{ end }}
 
+MOUNT {{ $ccachedir }}:/home/local/.ccache:Z
+
+{{ if eq .BUILD "android" }}
+RUN wget https://storage.googleapis.com/git-repo-downloads/repo \
+  && chmod u+x repo                                             \
+  && mkdir -p /home/local/.bin                                  \
+  && mv repo /home/local/.bin
+
+ENV PATH=/usr/lib/ccache:/home/local/.bin/:$PATH
+ENV USE_CCACHE=1
```

Comment:
I think we should move to the same point where the MOUNT for ccache is done:

ENV PATH=/usr/lib/ccache:$PATH
ENV USE_CCACHE=1

and leave only the /home/local/.bin and the /usr/lib/llvm-{{ $llvm_version }}/bin to the different branches of the "if"

Also, notice that the USE_CCACHE=1 is only exported in the Android branch (AFAIS)